File: BDNS Practical/Php/delete.php
<?php
    $m=new MongoClient();
    echo "Connection to database successfully";
    $db=$m->MYDB249613;
    echo "\n Database selected successfully";
    $col=$db->MyCol;
    echo "\nCollection selected successfully";

    $col->remove(array("name"=>"Bhupen"));
    echo "\nDocument removed successfully";
?>

File: BDNS Practical/Java/Delete.java
import com.mongodb.client.MongoCollection;
import com.mongodb.client.MongoDatabase;
import com.mongodb.MongoClient;
import com.mongodb.MongoCredential;
import com.mongodb.client.model.Filters;
import org.bson.Document;
 
public class Delete {
 
    public static void main(String[] args) {
        MongoClient mongo = new MongoClient("localhost", 27017);
        System.out.println("Connected to the database successfully");
 
       MongoDatabase database = mongo.getDatabase("TYIT51code");
        MongoCollection<Document> collection = database.getCollection("myCol");
 
        System.out.println("Collection myCol selected successfully");
 
        collection.deleteOne(Filters.eq("id", 1));
        System.out.println("Document deleted successfully");
    }
}